provisioning/templates-api/kubernetes/templates/etcd-values.yaml

```diff
@@ -1,6 +1,6 @@
 # See default values: https://github.com/bitnami/charts/blob/master/bitnami/etcd/values.yaml
 image:
-  tag: 3.5.4-debian-11-r27
+  tag: 3.5.5-debian-11-r16
```

Comment:
Ked to nasadzujeme znova, tak som rovno updatoval image.
(Usetri mi to jeden deploy, pokryvaju to testy.)